nautobot/dcim/tests/test_api.py

```diff
@@ -1381,9 +1381,7 @@ def setUpTestData(cls):
         interface_status = Status.objects.get_for_model(Interface).first()
 
         cls.devices = (
-            Device.objects.create(
-                device_type=cls.device_type, role=cls.device_role, name="Device 1", location=cls.location
-            ),
+            cls.device,
```

Comment:
Before this change we were actually creating two devices named `Device 1`, which is permitted for devices with a null `tenant`, but caused natural-key ambiguity errors.